### ISSUE TITLE
at the beginning of the training, total losses of many samples are close to 0

### ISSUE THREAD

**Issue by reeceshearsmith** [DetectionTeamUCAS/FCOS_Tensorflow#5]
I try to train my own dataset, and the loss of many samples is close to 0 all the time.  The final score is 10% lower than the pytorch version.  Can you help me?
Thank you.

**Comment by yangxue0827** [comment]
The latest code has been updated, and the performance has been greatly improved.
